zkvm/tests/zkvm.rs

```diff
@@ -5,12 +5,47 @@ use merlin::Transcript;
 
 use zkvm::*;
 
-fn issue_contract(
+fn predicate_helper(pred_num: usize) -> (Vec<Predicate>, Scalar, Predicate) {
+    let predicates = (0..pred_num)
+        .into_iter()
+        .map(|n| Predicate::from_signing_key(Scalar::from(n as u64)))
+        .collect();
+
+    // Generate issuance predicate
+    let issuance_pred = Predicate::from_signing_key(Scalar::from(100u64));
+    // Generate flavor scalar
+    let mut t = Transcript::new(b"ZkVM.issue");
```

Comment:
why not using the existing method? `Value::issue_flavor(&issuance_pred)`

Reply:
oh I forgot about the function, nice! updated